content/rs/administering/database-operations/migrate-to-active-active.md

```diff
@@ -0,0 +1,96 @@
+you'll need to create a new Active-Active database and migrate the data into the new database as a [Replica Of](https://docs.redislabs.com/latest/rs/administering/active-passive/) the existing database.
+This process will gradually populate the data in the Active-Active database.
+
+Before data migration starts, all data is flushed from the Active-Active database.
+The data is migrated to the Active-Active instance where you enabled Replica Of, and then the Active-Active replication copies the data from that instance to the other Active-Active instances.
+When data migration is finished, disable Replica Of and connect your applications to the Active-Active database.
+
+{{< note >}}
+During the migration, make sure that any applications that connect to the Active-Active database are **read-only**
+to make sure the dataset is identical to the source database during the migration process.
+{{< /note >}}
+
+To migrate an RS database to Active-Active:
+
+1. Create a new [Active-Active database]({{< relref "/rs/administering/creating-databases/create-active-active.md" >}}).
+
+    After the Active-Active database is activated, you see the database's configuration.
+
+1. Click **Edit** at the bottom of the database configuration.
+1. Enable **Migration using Replica Of**.
+1. Click **Continue** to confirm that you want to flush the data from the Active-Active database.
+1. Enter the URL of the source database endpoint:
+
+    <!-- Shared in create-active-passive.md -->
+
+    {{< note >}}
+The order of the Replica Of sources has no impact on replication.
+    {{< /note >}}
+
+    - For a source database in the same RS cluster - When you click on the box,
+    the available databases are shown in the correct format for the URL of the source endpoint:
+
+        ```sh
+        <database name>: redis://admin:<database_password>@<database_endpoint>:<database_port>
+        ```
+
+        You can select the database that you want to use as the source.
+
+    - For a source database in a different RS cluster:
+        1. Log in to the Web UI of the cluster that hosts the source database.
+        1. In **databases**, click on the database and go to **configuration**.
+        1. Under **Endpoint**, click on **Get Replica Of source URL**.
+
+            ![Replica Of source URL](/images/rs/replicaof-source-url.png)
+
+        1. Click **Copy to Clipboard** to copy the URL of the source endpoint.
+
+            If you want a different internal password, you can click **Regenerate Password**.
+
+            {{< warning >}}
+If you regenerate the password, replication to existing destinations fails until you update their configuration with the new password.
+            {{< /warning >}}
+
+        1. In the destination database, paste the URL of the source endpoint in the **Replica Of** box, and click ![Save](/images/rs/icon_save.png#no-click "Save").
+
+        {{< note >}}
+For a source database on a different RS cluster,
```

Comment:
Is there a reason you wouldn't want to compress the data and save bandwidth?

Reply:
there is default recommended level of compression already on :)

Reply:
They don't need to compress the data manually, since it's already done by default?